cmd/homi/setup/cmd.go

```diff
@@ -886,14 +890,21 @@ func writeValidatorsAndNodesToFile(validators []*ValidatorInfo, parentDir string
 	}
 }
 
-func writeTestKeys(parentDir string, keys []string) {
+func writeTestKeys(parentDir string, privKeys []*ecdsa.PrivateKey, keys []string) {
 	parentPath := path.Join(outputPath, parentDir)
 	os.MkdirAll(parentPath, os.ModePerm)
 
 	for i, key := range keys {
 		testKeyFilePath := path.Join(parentPath, "testkey"+strconv.Itoa(i+1))
 		ioutil.WriteFile(testKeyFilePath, []byte(key), os.ModePerm)
 		fmt.Println("Created : ", testKeyFilePath)
+
+		pk := privKeys[i]
```

Comment:
The length of `privKeys` might be different from the length of `keys`. 
They should be compared before this 'for loop'.

Reply:
I will make another PR for this. :)